### PR TITLE
[OpenMFP] OpenFGA SelfSubjectRulesReview implementation

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
With this PR, OpenFGA is queried to map FGA relations to RBAC resource rules.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: frontend/src/router/guards.js [hunk 1]
```diff
@@ -133,8 +133,11 @@ export function createGlobalResolveGuards () {
       }
 
       try {
+        const openMFP = useOpenMFP()
+        const accountId = openMFP.accountId.value ?? to.query.accountId
```
Comment (holgerkoser) [inline frontend/src/router/guards.js:137]:
```suggestion
        const { accountId } = useOpenMFP()
```

File: frontend/src/router/guards.js [hunk 2]
```diff
@@ -133,8 +133,11 @@ export function createGlobalResolveGuards () {
...
+
         const namespace = to.params.namespace ?? to.query.namespace
-        await refreshRules(authzStore, namespace)
+        await refreshRules(authzStore, namespace, accountId)
```
Comment (holgerkoser) [inline frontend/src/router/guards.js:140]:
```suggestion
        await refreshRules(authzStore, namespace, accountId.value ?? to.query.accountId)
```

File: frontend/src/router/guards.js (hunk 1 above)
Comment (petersutter) [reply frontend/src/router/guards.js]: ✅

File: frontend/src/router/guards.js (hunk 2 above)
Comment (petersutter) [reply frontend/src/router/guards.js]: ✅

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**